components/InputBox.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export interface InputBoxProps {
+  // eslint-disable-next-line prettier/prettier
+
+  /** Type of the input. */
+  inputType: string;
```

Comment:
We can specify the inputTypes in the form of strings or enums.

Reply:
Done